web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/CertificateMessageType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.service.facade.impl;
+
+public enum CertificateMessageType {
+    CERTIFICATE_ON_SAME_CARE_UNIT, CERTIFICATE_ON_SAME_DIFFERENT_CARE_UNIT, DRAFT_ON_SAME_CARE_UNIT, DRAFT_ON_SAME_DIFFERENT_CARE_UNIT,
```

Comment:
Should we remove the _SAME_ section from these enums?

- CERTIFICATE_ON_SAME_DIFFERENT_CARE_UNIT
- DRAFT_ON_SAME_DIFFERENT_CARE_UNIT